sageintacctsdk/sageintacctsdk.py

```diff
@@ -0,0 +1,81 @@
+"""
+Sage Intacct Python SDK
+"""
+from .exceptions import *
+from .apis import *
+
+class SageIntacctSDK:
+    """
+    Sage Intacct SDK
+    """
+
+    def __init__(self, sender_id: str, sender_password: str, user_id: str, company_id: str, user_password: str):
+        """
+        Initialize connection to Sage Intacct
+        :param sender_id: Sage Intacct sender id
+        :param sender_password: Sage Intacct sener password
+        :param user_id: Sage Intacct user id
+        :param company_id: Sage Intacct company id
+        :param user_password: Sage Intacct user password
+        """
+        # Initializing variables
+        self.__sender_id = sender_id
+        self.__sender_password = sender_password
+        self.__user_id = user_id
+        self.__company_id = company_id
+        self.__user_password = user_password
+
+        self.__access_token = None
+
+        self.api_base = ApiBase()
```

Comment:
is it necessary to expose `ApiBase`?
If yes why?

Reply:
dicussed;